modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java

```diff
@@ -138,8 +138,7 @@ else if (cmd.equals(Constants.UPDATE)) {
 		}
 
 		_assetDisplayPageEntryFormProcessor.process(
-			KBArticle.class.getName(), kbArticle.getKbArticleId(),
-			actionRequest);
+			KBArticle.class.getName(), kbArticle.getClassPK(), actionRequest);
```

Comment:
This cannot be right. The `KBArticleLayoutDisplayPageObjectProvider` and `KBArticleLayoutDisplayPageProvider` use the `kbArticleId`. Maybe there's an inconsistency somewhere else?

Reply:
We'll probably need to change everything to the class pk :-(